packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -339,25 +345,29 @@ def needs_dynamic_sidecar(self) -> bool:
         """if paths mapping is present the service needs to be ran via dynamic-sidecar"""
         return self.paths_mapping is not None
 
-    @validator("container_http_entry", always=True)
+    @field_validator("container_http_entry")
```

Comment:
always

Reply:
Added `validate_default=True` to `container_http_entry` field.